SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1116,6 +1111,23 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 	{
 		TArray<Worker_OpList*> OpLists = Connection->GetOpList();
 
+		// Servers will queue ops at startup until we've extracted necessary information from the op stream
+		if (bQueueOpsUntilReady)
+		{
+			if (!GlobalStateManager->ProcessOpListForServersCanBeginPlay(OpLists))
+			{
+				QueuedStartupOpLists.Append(OpLists);
+				return;
+			}
+
+			// The current TArray of OpLists contained the Ops the GSM needed to initiate startup.
+			// We can drain the startup queue and start processing ops now.
```

Comment:
order potentially matters here in terms of Worker_OpList processing.  I'll rework this